src/main/utils/mavenUtils.ts

```diff
@@ -73,6 +69,33 @@ export class MavenUtils {
         return [];
     }
 
+    /**
```

Comment:
Let's add a description for *nodeCanReachPom*.

Reply:
Adding JS Doc to `isPomReachable` function